generate.xml

```diff
@@ -116,7 +116,116 @@
+          <run test="limits_tests" />
+          <run test="stealth_tests" />
+          <!-- <run test="uint256_tests" /> -->
+          <!-- message -->
+          <run test="address_tests" />
+          <run test="alert_tests" />
+          <run test="alert_payload_tests" />
+          <run test="message_block_tests" />
+          <run test="block_transactions_tests" />
+          <run test="compact_block_tests" />
+          <run test="fee_filter_tests" />
+          <run test="filter_add_tests" />
+          <run test="filter_clear_tests" />
+          <run test="filter_load_tests" />
+          <run test="get_address_tests" />
+          <run test="get_block_transactions_tests" />
+          <run test="get_blocks_tests" />
+          <run test="get_data_tests" />
+          <run test="get_headers_tests" />
+          <run test="message_header_tests" />
+          <run test="headers_tests" />
+          <run test="heading_tests" />
+          <run test="inventory_tests" />
+          <run test="inventory_vector_tests" />
+          <run test="memory_pool_tests" />
+          <run test="merkle_block_tests" />
+          <run test="messages_tests" />
+          <run test="network_address_tests" />
+          <run test="not_found_tests" />
+          <run test="ping_tests" />
+          <run test="pong_tests" />
+          <run test="prefilled_transaction_tests" />
+          <run test="reject_tests" />
+          <run test="send_compact_tests" />
+          <run test="send_headers_tests" />
+          <run test="message_transaction_tests" />
+          <run test="verack_tests" />
+          <run test="version_tests" />
+          <!-- unicode -->
+          <run test="unicode_tests" />
+          <run test="unicode_istream_tests" />
+          <run test="unicode_ostream_tests" />
+          <!-- utility -->
+          <run test="binary_tests" />
+          <run test="collection_tests" />
+          <run test="data_tests" />
+          <run test="endian_tests" />
+          <run test="png_tests" />
+          <run test="random_tests" />
+          <run test="serializer_tests" />
+          <run test="stream_tests" />
+          <run test="thread_tests" />
+          <!-- wallet -->
+          <run test="bitcoin_uri_tests" />
+          <run test="ec_private_tests" />
+          <run test="ec_public_tests" />
+          <run test="electrum_tests" />
+          <!-- <run test="encrypted_keys_tests" /> -->`
+          <run test="hd_private_tests" />
+          <run test="hd_public_tests" />
+          <run test="message_tests" />
+          <run test="mnemonic_tests" />
+          <run test="payment_address_tests" />
+          <run test="qrcode_tests" />
+          <run test="select_outputs_tests" />
+          <run test="stealth_address_tests" />
+          <run test="stealth_receiver_tests" />
+          <run test="stealth_sender_tests" />
+          <run test="uri_tests" />
+          <run test="uri_reader_tests" />
+        </runner>
```

Comment:
This is bound to get us into trouble. Isn't there a blacklisting model as well?

Reply:
If you're willing to upgrade to boost 1.60 they've added an exclusion mechanism.  In fact, it looks as if you can (1) specify explicit tests to be run (2) specify tests which would default not be run to be added to the default set on a run and (3) exclude from the default set of tests some set of tests.  Our minimum support of 1.57 does not support this mechanism.

Reply:
Let's retain this as a future enhancement.